src/views/new-asset-view/layout.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { Link as RouterLink } from "react-router-dom";
+
+import { NewAsset } from "../../components/new-asset-form";
+import { locale } from "../../services";
+import { useUserFeedback } from "../../services/hooks/useUserFeedback";
+
+import { CreateNewAssetRequest } from "@mtfh/common/lib/api/asset/v1";
+import { ErrorSummary, Link, StatusBox } from "@mtfh/common/lib/components";
+
+export const NewPropertyLayout = (): JSX.Element => {
+  const [newProperty, setNewProperty] = useState<CreateNewAssetRequest | null>();
+
+  const {
+    showSuccess,
+    setShowSuccess,
+    showError,
+    setShowError,
+    errorHeading,
+    setErrorHeading,
+    errorDescription,
+    setErrorDescription,
+  } = useUserFeedback();
+
+  return (
+    <>
+      <Link as={RouterLink} to="#" variant="back-link">
+        Back
+      </Link>
+      <h1 className="lbh-heading-h1">New property</h1>
+
+      {showSuccess && (
+        <StatusBox variant="success" title={locale.assets.newPropertyAddedSuccessMessage}>
+          {
+            // Only include link if visible on MMH
+            (newProperty?.assetType === "Dwelling" ||
+              newProperty?.assetType === "LettableNonDwelling") && (
```

Comment:
I know I wrote this, but could this be replaced with a method? For example, `propertyTypeAllowed: (assetType: string) => boolean`

Reply:
I did that but in the end I move that whole 'link' section in a method called renderNewPropertyLink()